spsa.py

```
@@ -1,454 +1,422 @@
+
+    config["engines"][1]["cmd"] = args.engine
+    config["engines"][1]["options"] = options_minus
+
+    temp_config_path = "config.json"
+    with open(temp_config_path, "w", encoding="utf-8") as f:
+        json.dump(config, f, indent=4)
+
+    logger.info(f"--- Iteration {iteration}: {args.pairs} pairs ---")
+    cmd = ["./fastchess", "-config", f"file={temp_config_path}"]
+    try:
+        result = subprocess.run(cmd, capture_output=True, text=True, check=True)
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)